standards/request-response.md

```diff
@@ -504,7 +504,7 @@ SPS-CORS-Error: bad origin
 - A request without the presence of this header **MUST** indicate that the execution context is under the `production` dataflow and customer configuration, if applicable to this API.
```

Comment:
You will need to add a reference to every (or as many as make sense) code in the ruleset into the documentation: `<a name="sps-hosts-spscommerce-domain" href="#sps-hosts-spscommerce-domain"><i class="fa fa-check-circle" title="#sps-hosts-spscommerce-domain"></i></a>`  like here: https://github.com/SPSCommerce/sps-api-standards/blob/main/standards/url-structure.md

Reply:
I added around half of them. I excluded the ones about invalid response codes because it would look a little silly having all those icons under the summary table, and they weren't really referenced in the specific HTTP method sections.